src/views/admin/components/ApplicationSearch/ApplicationSearch.module.scss

```diff
@@ -0,0 +1,19 @@
+@import "styles/gallery-theme";
+@import "bootstrap/scss/_functions";
+@import "bootstrap/scss/_variables";
+@import "bootstrap/scss/mixins/_breakpoints";
```

Comment:
Chore [repeating]: we should use the Bootstrap utilities we have prepared, see comments on #26.